pysatCDF/tests/test_cdf.py

```diff
@@ -1,27 +1,23 @@
 import numpy as np
 
-from nose.tools import assert_raises, raises
-import nose.tools
-
 import pysatCDF
 
+
 class TestBasics:
     def setup(self):
         """Runs before every method to create a clean testing setup."""
-        # self.meta = pysat.Meta()
-        # self.testInst = pysat.Instrument('pysat', 'testing', tag='', clean_level='clean')
 
     def teardown(self):
         """Runs after every method to clean up previous testing."""
-        # del self.testInst
 
     def test_vefi_load(self):
         import os
```

Comment:
Probabaly want to clean this up by moving imports to top.  Might make debugging easier and could prevent double-import issues.